components/tests/ui/testcases/web/rdef_test.txt

```diff
@@ -148,9 +159,9 @@ Test Owners Rdef
     Go To                                   ${WELCOME URL}
     Select Experimenter
     Select And Expand Image
-    ${imageId}=                             Wait For Image Panel
+    ${imageId}=                             Wait For General Panel And Return Id      Image
     Click Link                              Preview
-    Wait For Preview Load
+    ${status}    ${oldId}    Wait For Preview Load      FAIL    '1'
```

Comment:
I wonder if it's possible to use the same assignment notation that I've used above ```${imageId}=``` , when you've got multiple assignments? E.g. ```${status}=    ${oldId}=``` since I find this much easier to understand.
Also, if you could maybe indent the  ``` Wait For Preview Load ``` to match indents above, this keeps it slightly neater (and do the same for similar lines below).